include/stream-client/detail/logger.ipp

```diff
@@ -0,0 +1,89 @@
+#pragma once
```

Comment:
This file should be put in a different folder. Only `detail` namespaces code goes into such `/detail` folder, usual `.ipp` files go into `/impl`